src/astbuilder.nit

```diff
@@ -102,6 +108,55 @@ class ASTBuilder
 	do
 		return new AIfExpr.make(condition, mtype)
 	end
+
+	# Make a new assert
+	fun make_assert(n_id : nullable TId , n_expr : AExpr , n_else : nullable AExpr): AAssertExpr
+	do
+		return new AAssertExpr.make(n_id , n_expr , n_else)
+	end
+
+	# Make a new method
+	fun make_method(n_visibility: nullable APublicVisibility,
+					tk_redef: nullable TKwredef,
+					mmethoddef: MMethodDef,
+					n_signature: nullable ASignature,
+					n_annotations: nullable AAnnotations,
+					n_extern_calls: nullable AExternCalls,
+					n_extern_code_block: nullable AExternCodeBlock,
+					n_block: AExpr): AMethPropdef
+	do
+		return new AMethPropdef.make(n_visibility, tk_redef, mmethoddef, n_signature, n_annotations, n_extern_calls, n_extern_code_block, n_block)
+	end
+
+	# Make a new or with two expr
+	fun make_or(right_expr: AExpr, left_expr: AExpr): AOrExpr
+	do
+		return new AOrExpr.make(right_expr,left_expr)
+	end
+
+	# Make a new and with two expr
+	fun make_and(right_expr: AExpr, left_expr: AExpr): AAndExpr
+	do
+		return new AAndExpr.make(right_expr,left_expr)
+	end
+
+	# Make a new parenthesis expr
+	fun make_parenthesis(expr: AExpr, annotations: nullable AAnnotations): AParExpr
```

Comment:
When do you need parenthesis ?

Reply:
Never but for a test I wanted to create parentheses. Since the code is done why remove it. But I agree that I do not know in what context this can be helpful.